ibis/sql/postgres/tests/test_functions.py

```diff
@@ -886,6 +887,16 @@ def test_rolling_window(alltypes, func, df):
     tm.assert_series_equal(result, expected)
 
 
+def test_rolling_window_with_mlb(alltypes):
+    with pytest.raises(NotImplementedError):
```

Comment:
Same comment re putting code outside the with.

Reply:
Fixed in next commit